docs/_articles/en/getting-started/java-setup.md

```diff
@@ -27,6 +27,8 @@ Inside Visual Studio Code, you will need to set the `salesforcedx-vscode-apex.ja
 
 MacOS:
 
+It is worth using the command 'Which Java' to find the correct path to your Java_Home.
```

Comment:
After taking a closer look, this is not accurate. `which java` does not confirm that a JDK is installed.

A better way of confirming the presence of a JDK is to run `/usr/libexec/java_home` to return the default JDK. If there's multiple installed versions, a user can run `/usr/libexec/java_home -V`.

I'll make the changes to the PR and get it merged.